cli.js

```diff
@@ -159,6 +197,7 @@ program.version(version, '-v, --version');
 program.command('build', 'Build your resume to the destination format(s).')
 	.argument('<source>', 'The path to the source JSON resume file.')
 	.option('-f, --format <format>', 'Set output format (HTML|PDF|YAML|DOCX|PNG|ALL)', formatOptionValidator, 'all')
+	.option('-s, --size <size>', 'Set output size for PDF files (A4|Letter|Legal|Tabloid|Ledger|A0|A1|A2|A3|A5|A6)', sizeOptionValidator, 'A4')
```

Comment:
Since the size actually only applies to PDF export we shall think about renaming the parameter to paper-size or pdf-size. Even puppeteer names it _Paper format_